esileapclient/common/utils.py

```diff
@@ -0,0 +1,56 @@
+import re
+import operator
+
+OPS = {
+    '>=': operator.ge,
+    '<=': operator.le,
+    '>': operator.gt,
+    '<': operator.lt,
+    '=': operator.eq,
+}
+
+
+def parse_property_filter(filter_str):
+    """Parse a property filter string into a key, operator, and value."""
+    match = re.match(r'([^><=]+)([><=]+)(.+)', filter_str)
+    if not match:
+        raise ValueError(f"Invalid property filter format: {filter_str}")
```

Comment:
If you do something like `--property "cpus==128"` then you get an uncaught error; can you catch it here?